### ISSUE TITLE
Text in Overlays is no longer selectable

### ISSUE THREAD

**Issue by mike-000** [openlayers/openlayers#9621]
**Describe the bug**
There's no obvious way to make text in Overlays selectable in OL6.  The `className `default in the documentation suggests they should default to selectable as they did in OL5.

**To Reproduce**
Compare 
https://openlayers.org/en/master/examples/overlay.html https://openlayers.org/en/master/examples/popup.html 
with 
https://openlayers.org/en/latest/examples/overlay.html https://openlayers.org/en/latest/examples/popup.html

**Expected behavior**
Text is selectable as in OL5 examples.

**Comment by webbbret2** [comment]
This appears to still be an issue. When you attempt to select the text in the overlay, the map moves.  Also, selection of the text still does not allow for selection. Without the overlay, I can select text, but the tooltip does not move with the map or when zooming. Has this issue been fixed and/or is there a workaround? Many thanks in advance.

**Comment by mike-000** [comment]
Although no-one has responded I have just looked again at the two sets of examples I linked and they are now behaving the same way - the text is selectable in both versions - something **has** changed between beta 9 and beta 10 which appears to fix my original issue.  You **will** need an overlay for text to move with the map, and it should now be selectable if using 6.0.0 beta 10 or 11.  Which version are you using?
